pydata_center/monitoring/views.py

```diff
@@ -273,6 +273,30 @@ def dashboard_view(request):
     )
 
 
+@api_view(['POST'])
+def create_agent_metric(request):
+    hostname = request.data.get('hostname')
+    if not hostname:
+        return Response({'error': 'Hostname is required'}, status=400)
+
+    try:
+        server_status = ServerStatus.objects.get(hostname=hostname)
+    except ServerStatus.DoesNotExist:
+        return Response(
+            {'error': f'Server with hostname {hostname} not found'},
+            status=404
+        )
+
+    data = request.data.copy()
+    data['server_status'] = server_status.id  # replace hostname with FK ID
+
+    serializer = AgentMetricSerializer(data=data)
+    if serializer.is_valid():
+        serializer.save()
+        return Response({'status': 'metric recorded'}, status=201)
```

Comment:
Unify the response format:
return Response({
    'status': 'metric recorded',
    'metric': serializer.data
}, status=201)